src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const repairedRobot = {};
+  const robotData = Object.entries(robot);
+
+  for (const data of robotData) {
+    const key = data[1];
+    const velue = data[0];
```

Comment:
use more semantic naming.

Reply:
or rewrite without using 1 and 0 indexes

Reply:
> use more semantic naming.

done